pygin/pygin/src/far_api.cpp

```diff
@@ -673,6 +684,11 @@ py::type far_api::type(std::string_view const TypeName)
 	return py::type(s_FarApi->m_Module[TypeName]);
 }
 
+[[nodiscard]] FILETIME far_api::file_time(py::object const& FileTimeInstance)
```

Comment:
It's not obvious at all what is going on here.
Maybe introduce a new type for Python date, same as with ints and floats and strings etc.?